build-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20.1-buster
+FROM golang:1.20.4-bullseye
 ARG goproxyValue
 ENV GOPROXY=${goproxyValue}
 RUN apt-get update && apt-get install -y curl python-requests python-yaml file jq unzip protobuf-compiler libprotobuf-dev && \
```

Comment:
```suggestion
RUN apt-get update && apt-get install -y curl file jq unzip protobuf-compiler libprotobuf-dev && \
```

Reply:
Installing `python-request` and `python-yaml` was causing error for `-bullseye` release. My quick research  tells me it's because `-bullseye (Debian 11)` deprecated python2, so we should use `python3-requests` instead. 

However .... I can't find any needs for python for Cortex's build process, so I removed `python-request` and `python-yaml`, then I was still able to successfully build a Cortex image. I suspect these 2 python packages was never needed, and was just copy and pasted from somewhere ?

Reply:
@rokh-conduktor let me know if you are ok with the change I suggested.

Reply:
python-requests was needed by a [scheduler](https://github.com/cortexproject/cortex/commit/3a63cf4ef2da9d5acc0de7b4a37585c4be0655fc#diff-fcce4108ea679f58257505f2830a5113ba2f3c37934edd2e14364b38da0cdccc) that no longer exists.

it should have been removed on https://github.com/cortexproject/cortex/pull/1616

Yes, this is fine, remove it

Reply:
Ohh I see, you dig deep into history @friedrichg! Thank you so much, now I can sleep better!